bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/AnomalousCouplings-SMP/ZZjj_llvvjj_ewk_aQGC/ZZjj_llvvjj_ewk_aQGC_proc_card.dat

```diff
@@ -0,0 +1,16 @@
+set group_subprocesses Auto
+set ignore_six_quark_processes False
+set loop_optimized_output True
+set complex_mass_scheme False
+
+import model QCKM_5_Aug21v2
+
+define p = g u c d s b u~ c~ d~ s~ b~
+define j = p
+define l+ = e+ mu+
+define l- = e- mu-
+define vl = ve vm vt
+define vl~ = ve~ vm~ vt~
+
+generate p p > z vl vl~ j j QCD=0 SMHLOOP=0 NP=1, z > l+ l-
```

Comment:
you don't need z -> tau tau, where tau decays to e , mu ?

Reply:
In our final state, after tau veto, we apply OSSF, delta_R_LL, Z_mass cut, etc., so the tau contribution is negligible after all the selections.